roles/openshift_facts/library/openshift_facts.py

```diff
@@ -393,6 +393,9 @@ def set_metrics_facts_if_unset(facts):
         if 'use_cluster_metrics' not in facts['common']:
             use_cluster_metrics = False
             facts['common']['use_cluster_metrics'] = use_cluster_metrics
+        if 'use_cluster_metrics_hawkular' not in facts['common']:
```

Comment:
@abutcher thoughts on persisting values like deploy_<hosted_role> for metrics, registry, router, etc that would allow for forcing creation even if there are no infra hosts defined and/or no storage defined?